package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "backblaze-b2",
-  "version": "1.4.0",
+  "version": "1.4.1",
```

Comment:
Looks good, but please keep the version the same as we have a process for releasing a new version.

Reply:
fixed